MxAnyType/CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+cmake_policy(SET CMP0048 NEW)
+project(MxAnyType LANGUAGES CXX VERSION 0.1)
+cmake_minimum_required(VERSION 3.0.2)
+
+# Configure the project
+# =====================
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+include(GNUInstallDirs)
+option(BUILD_SHARED_LIBS OFF)
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```

Comment:
This variable only make sense if you are building shared libraries.

Reply:
Yes, in the future this will become a shared library. For the time being it is static, but just con convenience.